helm/templates/client/job.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: batch/v1
+kind: Job
```

Comment:
i don't think it makes sense to create this as part of the helm chart.

if you want to include this in some sort of "usage examples" folder elsewhere, that's fine